SpringProject/src/test/java/com/gradproject2019/UserServiceTest.java

```diff
@@ -108,6 +116,50 @@ public void shouldThrowErrorWhenNewEmailExists() {
         userService.editUser(token.getToken(), userId, update);
     }
 
+    @Test(expected = UserForbiddenException.class)
+    public void shouldThrowErrorWhenRequestingManagerDoesNotMatchRequestedUser() {
+        given(authServiceImpl.getTokenById(token.getToken())).willReturn(token);
```

Comment:
Most of these givens are shared between tests, so they could be extracted into the setup method to help readability